src/main/java/team05/integrated_feed_backend/module/post/service/PostMapper.java

```diff
@@ -0,0 +1,46 @@
+package team05.integrated_feed_backend.module.post.service;
+
+import java.util.stream.Collectors;
+
+import team05.integrated_feed_backend.module.post.dto.response.HashtagDto;
+import team05.integrated_feed_backend.module.post.dto.response.PostDto;
+import team05.integrated_feed_backend.module.post.entity.Post;
+import team05.integrated_feed_backend.module.post.entity.PostHashtag;
+
+public class PostMapper {
+
+	public static PostDto toDto(Post post) {
+
+		if (post == null) {
+			return null;
+		}
+
+		return new PostDto(
+			post.getPostId(),
+			post.getTitle(),
+			post.getContent(),
+			post.getViewCount(),
+			post.getLikeCount(),
+			post.getShareCount(),
+			post.getCreatedAt(),
+			post.getUpdatedAt(),
+			post.getPostHashtags().stream()
+				.map(PostMapper::toDto)
+				.collect(Collectors.toList())
```

Comment:
자바 17에선 `.toList()` 를 바로 쓸 수 있어용~ 
변경하지 않으셔도 되구, 그냥 지식 공유였숩니당!
```suggestion
.toList()
```

Reply:
덕분에 배웠습니당 :) 감사합니다!